includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -311,27 +311,6 @@ private function courses_select( $courses, $selected_course, $select_id = 'cours
 		<?php
 	}
 
-	/**
-	 * Helper method to display the bulk action form in the modal.
-	 *
-	 * @param array $courses The courses options.
-	 */
-	private function render_bulk_actions_form( $courses ) {
-		$label = __( 'Select Course(s)', 'sensei-lms' );
-
-		?>
-		<form id="bulk-learner-actions-form" action="" method="post">
```

Comment:
There's a lot of orphaned code left behind in the codebase that reference elements or functions that were created or only called in this deleted function. It would be nice to clean all those up too.

Reply:
Here is the deprecated code removal and also since the change was connected in code I've fixed the issue with button not getting disabled when no bulk action is selected

https://github.com/Automattic/sensei/pull/5004/commits/c079f4a272b47b0402c7cb3195a2044a99033a05